docs/Concepts/Privacy/Multi-Tenancy.md

```diff
@@ -0,0 +1,26 @@
+description: Multi-tenancy
+<!--- END of page meta data -->
+
+# Multi-tenancy
+
+By default, each participant in a privacy network uses its own Besu and Orion
+node.
+
+Multi-tenancy allows multiple participants to use the same Besu and
+Orion node. Each participant is called a Tenant, and the Operator is the
```

Comment:
Each participant is called a Tenant -> Each participant is a Tenant

I think _called_ is redundant and it's also more parallel with the 2nd half of the sentence now.

Reply:
Done.